packages/lesswrong/server/twitterBot.ts

```diff
@@ -9,13 +8,14 @@ import { TwitterApi } from 'twitter-api-v2';
 import { getConfirmedCoauthorIds, postGetPageUrl } from "@/lib/collections/posts/helpers";
 import Users from "@/lib/vulcan-users";
 import { dogstatsd } from "./datadog/tracer";
+import { PublicInstanceSetting } from "@/lib/instanceSettings";
 
-const enabledSetting = new DatabaseServerSetting<boolean>("twitterBot.enabled", false);
-const karmaThresholdSetting = new DatabaseServerSetting<number>("twitterBot.karmaTreshold", 40);
-const apiKeySetting = new DatabaseServerSetting<string | null>("twitterBot.apiKey", null);
-const apiKeySecretSetting = new DatabaseServerSetting<string | null>("twitterBot.apiKeySecret", null);
-const accessTokenSetting = new DatabaseServerSetting<string | null>("twitterBot.accessToken", null);
-const accessTokenSecretSetting = new DatabaseServerSetting<string | null>("twitterBot.accessTokenSecret", null);
+const enabledSetting = new PublicInstanceSetting<boolean>("twitterBot.enabled", false, "optional");
+const karmaThresholdSetting = new PublicInstanceSetting<number>("twitterBot.karmaTreshold", 40, "optional");
```

Comment:
Note: these are actually private. Whether it is sent to the client is determined by whether it's in the "public" or "private" object in the settings file